src/types/PackageInfo.ts

```diff
@@ -24,7 +24,8 @@ export interface PackageInfo {
   devDependencies?: PackageDeps;
   peerDependencies?: PackageDeps;
   private: boolean;
-  options: PackageOptions;
+  combinedOptions: PackageOptions;
+  packageOptions: PackageOptions;
```

Comment:
nit: would be good to have comments for these here

Reply:
yuppers!